hwdb/20-vmbus-class.hwdb

```diff
@@ -0,0 +1,61 @@
+# This file is part of systemd.
+#
+# Data extracted from Linux hyperv.h
+
+vmbus:635161f83edfc546913ff2d2f965ed0e
```

Comment:
I think we should sort those alphabetically. This is generally done in other files of this type and it's the only option that doesn't lead to a complete mess in the long run.

We use single-space indentation in all hwdb files now. (The parser is happy with arbitrary number of spaces, but we use one everywhere for consistency.)